client/app/bundles/App/GettingStarted/Theming.md

```diff
@@ -1,3 +1,103 @@
 ## Theming
 
-We are currently updating all of our components to use CSS modules / CSS custom properties. More information on theming will be available after that work has been completed.
+Terra UI provides theming capabilities in various components via [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).
```

Comment:
Is there a reference as to which components provide theming capabilities?

Reply:
It would make sense to add "Theming" as an additional Component Feature for the components that are able to be themed.

Reply:
I'll see if I can use https://github.com/cerner/terra-core/blob/master/packages/terra-site/themeable-variables.json to list what components support theming. I also like the idea of adding to the list of component features.